use_case_examples/hybrid_model/compile_hybrid_llm.py

```diff
@@ -0,0 +1,140 @@
+"""Showcase for the hybrid model converter."""
+
+import os
+from copy import deepcopy
+from pathlib import Path
+from typing import List, Union
+
+import torch
+from concrete.fhe import Configuration, ParameterSelectionStrategy
+from datasets import load_dataset
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from concrete.ml.torch.hybrid_model import HybridFHEModel
+
+
+def compile_model(
+    model_name: str,
+    model: torch.nn.Module,
+    inputs: torch.Tensor,
+    module_names: Union[str, List],
+    expected_accuracy,
+    models_dir: Path,
+):
+    """Run the test for any model with its private module names."""
+
+    # Enable multi params/precision
+    configuration = Configuration(
+        single_precision=False,
+        parameter_selection_strategy=ParameterSelectionStrategy.MULTI,
```

Comment:
you can remove the MULTI part as it has been made default in CP now

Reply:
I'd rather have it explicit even if default.

Reply:
don't really agree as I believe we are going more and more towards not exposing strategies (meaning MULTI should be best and thus default, without new users being exposed to it), but I get your point, so if no one else pushed for this it's fine for me

Reply:
I agree with Roman here. I don't expect our users to understand what are these parameters (it's already hard for us to fully understand what they do). And there are more with the comparison options. I think this should all be handled behind the scene as the version of CML is enough to know what config is being used.

Reply:
Alright then I'll remove everything